clustered-indexes.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Clustered Indexes
+summary: Learn how clustered indexes apply to TiDB.
+---
+
+# Clustered Indexes
+
+> **Warning:**
+>
+> Clustered indexes is an experimental feature introduced in TiDB 5.0-rc.
```

Comment:
The clustered index is an experimental feature introduced in TiDB 5.0-rc. This document provides multiple examples to explain how this feature makes a difference to the query performance of TiDB. To enable this feature and see the detailed operation guide, see [tidb_enable_clustered_index
](/system-variables.md#tidb_enable_clustered_index-).